packages/@react-types/shared/src/dom.d.ts

```diff
@@ -136,17 +136,17 @@ export interface TextInputDOMProps extends DOMProps {
 
   // Selection events
   /**
-   * Handler that fires when some text in the input has been selected. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/select_event).
+   * Handler that is called when text in the input is selected. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/select_event).
    */
   onSelect?: ReactEventHandler<HTMLInputElement>,
 
   // Input events
   /**
-   * Handler that fires when the input value is about to be modified. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/beforeinput_event).
+   * Handler that is called when the input value is about to be modified. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/beforeinput_event).
```

Comment:
should we try to get rid of the words 'fires' in everything? because i found some more instances of it

Reply:
Mmm, good idea, I'll go ahead and do that

Reply:
actually I can't find anymore instances of `fires` in the types when I do a search

Reply:
https://github.com/adobe-private/react-spectrum-v3/search?q=fires&unscoped_q=fires

Reply:
I don't know if any of those can be gotten rid of, but those are all the instance i found
obviously we can't really get rid of it in `fireEvent`

Reply:
cool, the `fires` in `packages/@react-types/shared/src/dom.d.ts` should be handled in this pull, the others I will leave for now since they are comments